modules/videojsVideoProvider.js

```diff
@@ -25,29 +28,109 @@ export function VideojsProvider(config, videojs_, adState_, timeState_, callback
 
     // returns the player if it exists, or attempts to instantiate a new one
     player = videojs(divId, playerConfig, function() {
-        // callback runs in both cases
+      // callback runs in both cases
     });
+    // Todo: the linter doesn't like optional chaining is there a better way to do this
+    const vendorConfig = config.playerConfig && config.playerConfig.params && config.playerConfig.params.vendorConfig;
+    const tags = vendorConfig && vendorConfig.advertising && vendorConfig.advertising.tag;
```

Comment:
You can leave as is for now, the `try catch` approach might be a bit hacky. 
It might make sense for us instead to consider always populating these properties. We would have to make sure there isn't code checking for existence